libs/wingsdk/src/shared-aws/queue.inflight.ts

```diff
@@ -54,14 +55,25 @@ export class QueueClient implements IQueueClient {
   }
 
   public async pop(): Promise<string | undefined> {
```

Comment:
Was it tested on aws?

Reply:
yes, marked it as WIP to update the sdk test, it's in now